gradle.properties

```diff
@@ -55,3 +55,5 @@ junitVersion=4.12
 mockitoVersion=2.7.6
 slf4jVersion=1.7.21
 springVersion=3.2.9.RELEASE
+#set serverbase property here
+serverbase=
```

Comment:
Should this be coming from a different properties file? My understanding was that `gradle.properties` would be used for properties that can be used for all projects. `serverbase` is usually custom for each local environment.